gennav/planners/graph_search_algorithms/astar.py

```diff
@@ -0,0 +1,79 @@
+import os
+import math
+
+try:
+    from queue import PriorityQueue, Queue
+except ImportError:
+    from Queue import PriorityQueue, Queue
+
+
+def astar(graph, start, end, heuristic={}):
+    """Performs A-star search to find the shortest path from start to end
+    
+        Args:
```

Comment:
Can modify the docstrings to follow [this ](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) format. I think your matches pretty much apart from the type formatting.